src/main/java/domen/rideapp/infrastructure/AppConfig.java

```diff
@@ -38,4 +57,47 @@ RideService rideService(PricingService pricingService,
     DriverService driverService(DriverRepository repository) {
         return new DriverService(repository);
     }
+
+    @Bean
+    @ConfigurationProperties(prefix = "google.maps")
```

Comment:
W application.properties nie ma prefixu google.maps. Nawet jeśli używasz "prywatnie" innego profilu który ma te wartości, to w application.properties też powinny być, choćby puste